src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -209,7 +209,18 @@ MOBILE_INTEGRATION_TEST_TOKEN = '{{ MOBILE_INTEGRATION_TEST_TOKEN }}'
 
 ####### For Constructing Global URLs ########
 
-LOCAL_APPS = ('dimagi.utils', 'gunicorn',)
+LOCAL_APPS = (
+    'dimagi.utils',
+    'gunicorn',
+    {% for app_name, host in localsettings.get('LOCAL_APPS', {}).items() %}
+    {% if host == inventory_hostname %}
```

Comment:
I think that inventory_hostname would be the machines name. like hqdjango1. what you probably want is something like inventory_groups (i don't know if that exists or not) and check `if inventory_groups.instersect(host)`